src/utils/utils.tsx

```diff
@@ -61,10 +61,19 @@ export const asArray = (objectOrArray) => {
   return [objectOrArray];
 };
 
-const getModifiedList = <T extends { name: string }>(items: T[], selected: string | null): T[] => {
+/**
+ * Return reordered list of entities so that selected entity will become first,
+ * while order of the remaining entities won't change.
+ * In case there is no selected entity specified, return the original list.
+ * Entities are identified by attribute `iri`.
+ *
+ * @param items List of items.
+ * @param selected Selected item or null.
+ */
+const reorderEntities = <E extends { iri?: string }>(items: E[], selected: E | null): E[] => {
```

Comment:
I suggest getReorderedEntities, see comment above

Reply:
renamed to "getReorderedEntities"